### PR TITLE
testing info + typo

### PR DESCRIPTION
**Pull request by ramabile**
Additional information if one wants to test their own installation.

### CODE REVIEW

File: INSTALL.rst [hunk 1]
```diff
@@ -76,3 +76,21 @@ To use debug information in PySCIPOpt you need to build it like this:
...
+
+Testing new installation
+========================
+
+To test your brand-new installation of PySCIPOpt you need `pytest <https://pytest.org/>`__ on your system:
+
+::
+
+    sudo apt-get install python-pytest   # for Python 2, on Linux
+    sudo apt-get install python3-pytest  # for Python 3, on Linux
```
Comment (fserra) [inline INSTALL.rst]: I would link to the install information on pytest instead, especially if the suggested instructions require `sudo`

**Comment by fserra** [comment]
Thanks!

File: INSTALL.rst [hunk 2]
```diff
@@ -44,7 +44,7 @@ your ``PATH`` environment variable:
 
 On Linux and OS X this is encoded in the generated PySCIPOpt library and therefore not necessary.
 
-Building everything form source
+Building everything from source
```
Comment (mattmilten) [inline INSTALL.rst]: Thanks :-)

File: INSTALL.rst (hunk 1 above)
Comment (mattmilten) [inline INSTALL.rst]: What about `pip install pytest`? This does not require admin permissions.

File: INSTALL.rst [hunk 3]
```diff
@@ -76,3 +76,21 @@ To use debug information in PySCIPOpt you need to build it like this:
...
+
+::
+
+    sudo apt-get install python-pytest   # for Python 2, on Linux
+    sudo apt-get install python3-pytest  # for Python 3, on Linux
+
+Tests are in the ``tests`` directory. Any test can be run with:
+::
+
+   python -m pytest test_name.py
```
Comment (mattmilten) [inline INSTALL.rst]: It's easier to run `py.test` or `py.test tests/test_*.py` to run only a specific test.

**Pull request merged**